pandas/tests/io/test_parquet.py

```diff
@@ -567,6 +568,24 @@ def test_s3_roundtrip_for_dir(self, df_compat, s3_resource, pa, partition_col):
             repeat=1,
         )
 
+    @tm.network
+    @td.skip_if_no("pyarrow")
+    def test_parquet_read_from_url(self, df_compat):
+        # TODO:alimcmaster1 update with master URL
+        url = (
+            "https://raw.githubusercontent.com/alimcmaster1/pandas/"
```

Comment:
This might fail due to rate limits from github?

Reply:
Yep fair point - we already do this in test_network.py and I think the decorator helps handle any failures. We could use https://pypi.org/project/pytest-localserver/ ? also I couldnt find docs that suggest what the rate limits are for raw.githubusercontent endpoints?